draft-irtf-cfrg-vdaf.md

```diff
@@ -379,54 +379,264 @@ exception that Clients submitting measurements may be anonymous.
 
 # Definition of DAFs {#daf}
 
-> TODO Issue#20
+By way of a gentle introduction to VDAFs, this section describes a simpler class
+of schemes called Distributed Aggregation Functions (DAFs). Unlike VDAFs, DAFs
+do not provide verifiability of the computation. Clients must therefore be
+trusted to compute their input shares correctly. Because of this fact, the use
+of a DAF is NOT RECOMMENDED for most applications. See {{security}} for
+additional discussion.
+
+A DAF scheme is used to compute a particular "aggregation function" over a set
+of measurements generated by Clients. An aggregation function may include an
+optional "aggregation parameter", which is selected by the Collector and
+disseminated among the Aggregators. The semantics of this parameter is specific
```

Comment:
"The semantics of this parameter *are* specific"

(GitHub won't let me use the suggestion feature when I review commit-by-commit)

Reply:
Hmm, TIL the semantics of the word "semantics". Well, maybe the IETF has a style guide that someone can cite from to settle this. For now I say go with whichever you prefer.